src/users/role.go

```diff
@@ -31,7 +31,7 @@ var defaultRoleTemplates = map[string]UserInfo{
 		Cape: "https://files.impactclient.net/img/texture/staff_cape.png",
 		Editions: []Edition{{
 			Text: "Staff",
-			TextColor: "PURPLE",
+			TextColor: "#FF9542F5",
```

Comment:
gofmt aligns values in structs. You need to put enough spaces in front of "Staff", for it to be aligned with "#FF9542F5"